exercises/leap/src/main/scala/Leap.scala

```diff
@@ -0,0 +1,3 @@
+object Year {
```

Comment:
@abo64 As far as I can recall: yes, you are right. I think reviewing the other exercises makes sense.